usagov/usagov-2021
# USAGOV-1467: Account for parameters for a route that have type starting with "entity:" that does not relate to an entity

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1467

## Description

Check that $entity is defined before calling its methods in usa_translation.module

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

Visit http://localhost/filter/tips . You should see some text. If you're logged in, you'll see quite a lot of text. If you do this on the cms without this fix, you'll see a server error instead. 

That page is what opens when you click the "About text formats" link under the rich text editor in the CMS. 

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
